works.html

```diff
@@ -71,7 +71,300 @@
         </div>
       </header>
       <main class="main">
-        <!-- Start your work here ... -->
+        <div class="main__global-container">
+          <div class="main__global-container-title">
+            <h2 class="title-heading">Буровые работы</h2>
+            <p class="main__global-container-title-text">
+              МФЦА – это уникальный центр, который использует лучшие практики и
+              подходы международных финансовых центров и сотрудничает с крупными
+              корпорациями и небольшими компаниями, недавно появившимися на рынке.
+            </p>
+          </div>
+          <div class="main__wrapper">
+            <div class="main__wrapper-common">
+              <img
+                class="main__wrapper-common-picture"
+                src="./assets/images/works/drilling-works.png"
+                alt="drilling-works"
+              />
+              <div class="main__wrapper-common-text">
+                <h3 class="common-text-heading">Буровые работы</h3>
```

Comment:
Тот же вопрос, что и выше. Я бы сделал это элементом.
```suggestion
                <h3 class="main__common-text-heading">Буровые работы</h3>
```